### PR TITLE
2022/3/3

### PR DESCRIPTION
**Pull request by mksmkss**
test

### CODE REVIEW

**Comment by Jun-Ikeda** [comment]
hello

**Comment by Jun-Ikeda** [comment]
ano fairu ga okashii desuyo

File: dev/demo.js [hunk 1]
```diff
@@ -3,28 +3,37 @@ import React, { Component } from 'react';
...
+
 const style = StyleSheet.create( {
     container: {    
         flex: 1,
         justifyContent: 'center',
         alignItems: 'center',
     },
-
+    con: {
+        alignItems: 'center',
```
Comment (Jun-Ikeda) [inline dev/demo.js:14]: Please add justifyContent: 'center',

Comment (mksmkss) [reply dev/demo.js:14]: ok

**Pull request merged**